### DISCUSSION TITLE
Anderson bale collector - unload handeling

### DISCUSSION THREAD

**Discussion by rento112** [Courseplay/courseplay#6948] [Q&A] [answered]
Atleast when it comes to the bale trailers that unload almost horizontally like the the round balers in the DLCs, Courseplay "finishes" and returns to the normal mode of the trailer instead of continuing to push the round bales onto the loading point. Maybe set a timer for that perhaps? It makes it where alot of the bales remain on the trailer.

**Comment by Tensuko** [comment, marked as answer]
You can't solve everything with timers...
That would be a vehicle adjustment to handle those bale collectors in a special way.

**Reply by rento112** [reply]
I cant say anything about that since I dont know programming, but wouldnt it be simpler to make it where it relies on a timer  for when things finish? Atleast as a secondary option to be enabled by the user when the program itself cant do it reliably?

ALso, got any advice on how to make it where my bale collector doesnt unload too early? I think it's registering my unloading point as obstructed or something and unload earlier in  the way points even though it drives through just fine.
